site/guide/manage-groups.qmd

```diff
@@ -0,0 +1,50 @@
+---
+title: "Manage groups"
+keywords: "manage groups, group management, ValidMind"
+---
+
+This task involves managing user groups within ValidMind, allowing for organized access control.
+
+## Prerequisites
+
+To perform these steps, you must be a group administrator or a superuser. 
+
+## Add new groups
+
+1. [Log in to the ValidMind UI]({{< var vm_ui >}}).
+
+2. Under **User Permissions** in the sidebar, select **Groups**.
+
+3. Click on the "Add New Group" button to initiate the group creation process.
+   - Provide a name for the new group.
+   - Optionally, specify a description for the group's purpose.
+   - Add users to the group by selecting them from a list or specifying their usernames.
+   - Save the new group configuration.
```

Comment:
```suggestion
   
4. Click **WHATEVER_THE_BUTTON_IS_CALLED** to save the new group configuration.
```

This needs to be its own step (or the bulleted list items under the previous step need to be numbered/lettered — bullet list items can be performed in any order). Please also renumber the following steps. 

Note I don't see the option to create a group, so you will also need to update WHATEVER_THE_BUTTON_IS_CALLED. 🙂 I think it makes sense to call out the specific UI control here to mark the end of the previous step.

Reply:
Can we hold on updating the Manage Groups as this is still work in progress? Let me work more closely with the developers on completing this bit and I can update the content better

Reply:
Concluded that main changes will not include this page.